docs/guides/latest-updates.mdx

```diff
@@ -29,17 +29,28 @@ Jump to [Qiskit SDK](#latest-qiskit-sdk) | [Qiskit Runtime client](#latest-qiski
 
 This section summarizes the recent enhancements and new features for the new [IBM Quantum Platform.](https://quantum.cloud.ibm.com/)
 
+**5 June 2025**
+
+- Added instance job filter: The platform now displays only your workloads by default. On the dashboard, you will see only the latest workloads you have submitted to instances within your account.
+
+![Only the latest workloads now shown by default](/docs/images/guides/latest-updates/workloads.avif)
+
+- A new provider release, featuring:
```

Comment:
```suggestion
- A new qiskit-ibm-runtime release, featuring:
```

We sometimes call it `provider` because it was originally `qiskit-ibmq-provider`, but it's been `qiskit-ibm-runtime` for many years now.